sysutils/duplicity/Portfile

```diff
@@ -1,41 +1,40 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
 
 PortSystem          1.0
 PortGroup           python 1.0
 
 name                duplicity
 categories          sysutils
 
-version             0.7.17
+version             0.8.15          ;# released July 27, 2020.
 
 set stable_series   [join [lrange [split ${version} .] 0 1] .]-series
 platforms           darwin
 license             GPL-2
 maintainers         nomaintainer
 
-description   Encrypted bandwidth-efficient backup.
+description         Encrypted bandwidth-efficient backup.
+long_description    Duplicity backs up directories by producing encrypted \
+                    tar-format volumes and uploading them to a remote or local file \
+                    server. Because duplicity uses librsync, the incremental archives \
+                    are space efficient and only record the parts of files that have \
+                    changed since the last backup. Because duplicity uses GnuPG to \
+                    encrypt and/or sign these archives, they will be safe from spying \
+                    and/or modification by the server.
 
-long_description  Duplicity backs up directories by producing encrypted \
-      tar-format volumes and uploading them to a remote or local file \
-      server. Because duplicity uses librsync, the incremental archives \
-      are space efficient and only record the parts of files that have \
-      changed since the last backup. Because duplicity uses GnuPG to \
-      encrypt and/or sign these archives, they will be safe from spying \
-      and/or modification by the server.
-
-distname            duplicity-${version}
+distname            ${name}-${version}
 homepage            http://duplicity.nongnu.org/
 master_sites        https://launchpad.net/duplicity/${stable_series}/${version}/+download
 
-checksums           rmd160  20d6e1d63d3da925009709cafd3e721af68bcc2e \
-                    sha256  3724c5f1f839e584e49154ee0ff137e6f3450eedbd7f3886f31d2093001cb04a \
-                    size    1719145
+checksums           rmd160  bbd3dcc5a067cfadfe067af1f90abdf2565b0b8f \
+                    sha256  ac69ff68b88b6c15c222f8334bb447fbe529c6849a6c9ac33e4995d7eb31e4cd \
+                    size    1482647
 
-python.default_version 27
+python.default_version 38
```

Comment:
Build fails:
```
Error: Dependency 'py38-rbtools' not found.
```

Reply:
Ah sorry, I see it depends on your other PRs #8034 and #8035.

Reply:
I wasn't sure if I could mash all into one PR, so I assumed the rule was 1 PR per port.